### PR TITLE
Nerfs Rathen's Secret and Magic Missile spells

### PR DESCRIPTION
**Pull request by Lythine**
[ADD TO WIKI] [BALANCE]

## About the PR 

Removes weakened from Rathen's Secret spell.
Changes weakened time on Magic Missile from 5 to 4. (3 to 2 on the weak projectile.)

## Why's this needed? 

Recently, Rathen's Secret has been picked extremely often due to how well it synergizes with Magic Missile and with good reason. Nearly every wizard round, at least one wizard would pick this combo and search for players, then use Rathen's followed by Magic Missile. This would nearly always put all nearby players in a 6 second unavoidable stun which could be swiftly followed up with one of the other spells, such as Prismatic Spray, Empower, Polymorph, Clown's Revenge, etc. This makes it extremely easy for just one wizard to slowly kill a majority of the station with little to no counterplay. This PR aims to remove this broken combo by turning Rathen's into a damage-focused spell instead of a stun as well as slightly decreasing how long you're stunned for after getting hit by Magic Missile so it isn't as much as a free kill every time it's cast.

## Changelog

```changelog
(u)Lythine
(*)Rathen's Secret no longer weakens. Magic Missile's weaken time slightly shortened.
```

### CODE REVIEW

**Review by ZeWaka — COMMENTED** [review]
code is correct

**Comment by flappybatpal** [comment]
This will be controversial. I hereby punish you with a forum thread!!!

File: code/modules/events/gimmick/bigfart.dm [hunk 1]
```diff
@@ -91,7 +91,8 @@
 			ThrowRandom(B, dist = 6, speed = 1)
 		H.visible_message("<span class='alert'><b>[H]</b>'s [magical ? "arse" : "ass"] tears itself away from \his body[magical ? " in a magical explosion" : null]!</span>",\
 		"<span class='alert'>[changer ? "Our" : "Your"] [magical ? "arse" : "ass"] tears itself away from [changer ? "our" : "your"] body[magical ? " in a magical explosion" : null]!</span>")
-		H.changeStatus("weakened", 2 SECONDS)
+		if (!magical)
+			H.changeStatus("weakened", 2 SECONDS)
 		severed_something = TRUE
 		H.force_laydown_standup()
```
Comment (Sovexe) [inline code/modules/events/gimmick/bigfart.dm:97]: probably shouldn't be calling `force_laydown_standup()` if you aren't using it to apply weakness

File: code/modules/events/gimmick/bigfart.dm [hunk 2]
```diff
@@ -157,9 +158,7 @@
 		boutput(H, "<span class='notification'>[changer ? "We" : "You"] hear an otherworldly force let out a short, disappointed cluck at [changer ? "our" : "your"] lack of an arse.</span>")
 	H.visible_message("<span class='alert'>[is_bot ? "Oily chunks of twisted shrapnel" : "Wadded hunks of blood and gore"] burst out of where <b>[H]</b>'s [magical ? "arse" : "ass"] used to be!</span>",\
 	"<span class='alert'>[nobutt_phrase[assmagic]]</span>")
-	if(magical)
-		H.changeStatus("weakened", 1 DECI SECOND)
-	else
+	if(!magical)
 		H.changeStatus("weakened", 3 SECONDS)
 	H.force_laydown_standup()
```
Comment (Sovexe) [inline code/modules/events/gimmick/bigfart.dm:163]: probably shouldn't be calling `force_laydown_standup()` if you aren't using it to apply weakness

**Review by Sovexe — COMMENTED** [review]
Not sure about these changes. Anyways...

**Pull request merged**